chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+if ! [[ -r `pwd` ]]; then
+    echo "Directory is not readable."
+    exit 255
+elif [[ "$#" -lt 2 ]]; then
+    echo "At least 2 args are expected"
+    exit 1
+elif [[ "$#" -gt 2 ]]; then
+    echo "Too many args, only 2 args are expected"
+    exit 1
+elif [[ -e $1 && -f $2 ]]; then
```

Comment:
It looks rather strange, when you check 2 files existence with different approaches (for 1st file you use `-e` and for 2d `-f` checks). For other people reading your code it would be very confusing and may lead to different code understanding later